cabal.project

```diff
@@ -120,26 +92,6 @@ allow-newer:
 -- --------------------------- 8< --------------------------
 -- Please do not put any `source-repository-package` clause above this line.
 
-source-repository-package
-  type: git
-  location: https://github.com/input-output-hk/cardano-node
```

Comment:
This changes the pinned cardano-node version to 1.35.3 though right? My idention was to use node 1.35.4 and the same version dependencies as node 1.35.4. I think this cannot be achived simply by the index-state date, can it?

Reply:
You can just use normal version bounds to get that, I think.